app/javascripts/tiles/downloads.jsx

```diff
@@ -160,11 +182,11 @@ var DownloadsTile = React.createClass({
         actualdataset.push(null);
       }
 
-      var lastThreeMonths = (maindataset[maindataset.length - 1] + maindataset[maindataset.length - 2] + maindataset[maindataset.length - 3]);
-      var prevThreeMonths = (maindataset[maindataset.length - 4] + maindataset[maindataset.length - 5] + maindataset[maindataset.length - 6]);
+      var lastSixMonths = (maindataset[maindataset.length - 1] + maindataset[maindataset.length - 2] + maindataset[maindataset.length - 3] + maindataset[maindataset.length - 4] + maindataset[maindataset.length - 5] + maindataset[maindataset.length - 6]);
+      var prevSixMonths = (maindataset[maindataset.length - 7] + maindataset[maindataset.length - 8] + maindataset[maindataset.length - 9] - maindataset[maindataset.length - 10] + maindataset[maindataset.length - 11] + maindataset[maindataset.length - 12]);
 
-      var totalGrowthInDownloads = lastThreeMonths - prevThreeMonths;
-      var growth = Math.round(Math.abs(totalGrowthInDownloads) / prevThreeMonths * 100);
+      var totalGrowthInDownloads = lastSixMonths - prevSixMonths;
+      var growth = Math.round(Math.abs(totalGrowthInDownloads) / prevSixMonths * 100);
```

Comment:
We have a lot more data than we used to, and six months will show a consistently more favorable picture, given the ebbs and flows of crypto dev